src/main/java/com/template/entity/Player.java

```diff
@@ -0,0 +1,37 @@
+package com.template.entity;
+
+import jakarta.persistence.*;
+import jakarta.validation.constraints.NotEmpty;
+import jakarta.validation.constraints.Size;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import java.time.LocalDateTime;
+import java.util.Objects;
+
+@Data
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+@Entity
+@Table(name = "player",
+        uniqueConstraints = {
+                @UniqueConstraint(columnNames = "username")
+        })
+public class Player {
+
+  @Id
+  @GeneratedValue(strategy = GenerationType.IDENTITY)
+  private Long playerId;
+
+  @NotEmpty(message = "Username should not be empty.")
+  @Size(min = 5, max = 50, message = "The length of the username must be from 5 to 50 characters inclusive.")
+  private String username;
+
+  private Integer eloRating = 1000;
```

Comment:
There's already exists default value `1000` for this field in the CREATE TABLE statement, remove this logic from java code